CommonTools/RecoAlgos/interface/PrimaryVertexAssignment.h

```diff
@@ -82,6 +98,31 @@ class PrimaryVertexAssignment {
     return chargedHadronVertex(
         vertices, reco::TrackRef(), &(*pfcand.gsfTrackRef()), time, timeResolution, jets, builder);
   }
+
+  std::pair<int, PrimaryVertexAssignment::Quality> chargedHadronVertex(const reco::VertexCollection& vertices,
+                                                                       const pat::PackedCandidate& pfcand,
+                                                                       const edm::View<reco::Candidate>& jets,
+                                                                       const TransientTrackBuilder& builder) const {
+    float time = 0, timeResolution = -1;
+    if (useTiming_ && pfcand.timeError() > 0) {
+      time = pfcand.time();
+      timeResolution = pfcand.timeError();
+    }
+    if (!pfcand.hasTrackDetails())
+      return std::pair<int, PrimaryVertexAssignment::Quality>(-1, PrimaryVertexAssignment::Unassigned);
```

Comment:
```suggestion
      return {-1, PrimaryVertexAssignment::Unassigned};
```
could perhaps improve readability?